client/components/theme-type-badge/test/index.jsx

```diff
@@ -7,6 +7,13 @@ import { Provider } from 'react-redux';
 import configureStore from 'redux-mock-store';
 import ThemeTypeBadge from '../';
 
+jest.mock( 'calypso/state/sites/hooks/use-site-global-styles-status', () => ( {
+	useSiteGlobalStylesStatus: () => ( {
+		shouldLimitGlobalStyles: false,
+		globalStylesInPersonalPlan: false,
+	} ),
```

Comment:
Nitpick: `shouldLimitGlobalStyles` probably doesn't need to be mocked
```suggestion
	useSiteGlobalStylesStatus: () => ( {
		globalStylesInPersonalPlan: false,
	} ),
```